lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -60,15 +66,91 @@ def diode_params_from_voltage(
     )
 
 
-def viscosity_of_water(temperature):
-    """Computes the viscosity of water in [Pa*s] at a particular temperature.
+def density_of_water(temperature, molality, pressure=0.101325):
+    """Determine the density of water with NaCl.
+
+    This model is based on [1]_.
+
+    Parameters
+    ----------
+    temperature : array_like
+        Temperature in C
+    molality : float
+        Molality NaCl [mol/kg]
```

Comment:
Yeah, it makes sense. I'd prefer to not use 1000 for the density of water (since we have an explicit model of that too), but we can convert exactly using some basic rootfinding. I'll add it in a separate commit.